admin-dev/themes/new-theme/js/pages/order/view/order-product-add-autocomplete.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+import Router from '../../../components/router';
+
+const $ = window.$;
+
+export default class OrderProductAutocomplete {
+  constructor(input) {
+    this.router = new Router();
+    this.input = input;
+    this.results = {};
+    this.dropdown = $('<div class="dropdown"></div>');
+    this.dropdownMenu = $('<div class="dropdown-menu"></div>');
+    this.dropdown.html(this.dropdownMenu);
+    this.input.after(this.dropdown);
+    this.onItemClickedCallback = () => {};
+  }
+
+  listenForSearch() {
+    this.input.on('click', (event) => {
+      event.stopImmediatePropagation();
+      this.updateResults(this.results);
+    });
+    this.input.on('keyup', event => this.search(event.target.value));
+    $(document).on('click', () => this.dropdownMenu.hide());
```

Comment:
No, after tests, If i put a blur event on this.input, that correctly hide the dropdown menu, but the click on item is not propagated to item, because the blur is passed before.